index.html

```diff
@@ -32,6 +32,16 @@
             src="img/QuestionPhoto.png"
             alt="image for quiz question"
           />
+          <div id="setGameLevel">
```

Comment:
nie używałbym czasowników dla nazw klass/id css
+ trzeba sformatować kod, jakieś niepotrzebne spacje są poniżej

Reply:
Planujemy odejść od pisania kodu w HTML, fajnie by było zrobić na to component który będzie zmieniał gameLevel w ramach config. Serwis dostanie ten config do działania i w ten sposób nie będzie musiał nic wiedzieć o drzewie DOM.

Reply:
Ale zgadzam się z Marcinem, GameLevelSelector jest lepszą nazwą dla tej sekcji/przyszłego komponentu

Reply:
thx, dałam GameLevel + wyrzuciłam spacje

Reply:
Dorzuciłam jako oddzielny komponent